volmaster/daemon.go

```diff
@@ -71,25 +71,33 @@ func (d *DaemonConfig) Daemon(listen string) {
 	r := mux.NewRouter()
 
 	postRouter := map[string]func(http.ResponseWriter, *http.Request){
-		"/request":      d.handleRequest,
-		"/create":       d.handleCreate,
-		"/copy":         d.handleCopy,
-		"/mount":        d.handleMount,
-		"/mount-report": d.handleMountReport,
-		"/unmount":      d.handleUnmount,
-		"/remove":       d.handleRemove,
+		"/global":                           d.handleGlobalUpload,
+		"/volumes/create":                   d.handleCreate,
+		"/volumes/copy":                     d.handleCopy,
+		"/volumes/remove":                   d.handleRemove,
+		"/volumes/removeforce":              d.handleRemoveForce,
+		"/volumes/request":                  d.handleRequest,
+		"/mount":                            d.handleMount,
```

Comment:
should `mount` and `unmount` be part of `volumes` hierarchy? i.e. `volumes/mount/` and `volumes/unmount/`